b2b-sample/guardio-insurance-sample-application/util/apiCall/settings/callSwitchOrg.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import config from '../../../config.json';
+import { getInternalApiRequestOptionsWithParam } from '../../util/apiUtil/getInteralApiRequestOptions';
+
+export default async function callSwitchOrg(subOrgId, accessToken) {
+
+    try {
+        const res = await fetch(
+            `${config.WSO2IS_CLIENT_URL}/api/settings/switchOrg`,
+            getInternalApiRequestOptionsWithParam(null, subOrgId, accessToken)
+        );
+
+        if (res.status != 200) {
+
```

Comment:
Do we need these new lines after this if statements? Check other places as well.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/130d957728499e5e0e731eba0b663086f931fe95